tests/meta/upload/scout/test_meta_upload_scoutapi_rna.py

```diff
@@ -590,9 +590,10 @@ def test_add_rna_sample(
     )
 
     # THEN the resulting RNADNACollections should contain all RNA samples in the case
+    assert rna_sample_list
     for sample in rna_sample_list:
         assert sample.internal_id in [
-            rna_dna_collection.rna_sample_id for rna_dna_collection in rna_dna_collections
+            rna_dna_collection.rna_sample_internal_id for rna_dna_collection in rna_dna_collections
```

Comment:
The model attribute `rna_sample_id` has been renamed. The test did not fail because it was iterating through an empty list, since the like search did not include `%`